examples/vision/probing_vits.py

```diff
@@ -0,0 +1,679 @@
+"""
+Title: Under the Hood of Vision Transformers
```

Comment:
Let's find a better title. "Explainability for Vision Transformers" / "Investigating Vision Transformers representations" / something like that.